common/txmgr/txmgr.go

```diff
@@ -241,14 +242,14 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Close() (m
 		}
 		if b.fwdMgr != nil {
 			if err := b.fwdMgr.Close(); err != nil {
-				merr = errors.Join(merr, pkgerrors.Wrap(err, "Txm: failed to stop ForwarderManager"))
+				merr = errors.Join(merr, fmt.Errorf("Txm: failed to stop ForwarderManager: %s", err))
```

Comment:
We may want to use the `%w` verb for errors so `err` gets wrapped instead of stringified. I think that'd maintain the current functionality in case we're using `errors.Is` or `errors.As` upstream. I don't want to overload the comments so only commenting here but I think this can be changed for every `fmt.Errorf` updated in this PR